utils/wallet/provider/uphold/uphold.go

```diff
@@ -953,7 +968,7 @@ func FundWallet(destWallet *Wallet, amount decimal.Decimal) (decimal.Decimal, er
 		return zero, errors.New("donor wallet does not have an ID")
 	}
 
-	_, err = donorWallet.Transfer(altcurrency.BAT, altcurrency.BAT.ToProbi(amount), destWallet.Info.ProviderID)
+	_, err = donorWallet.Transfer(altcurrency.BAT, amount, destWallet.Info.ProviderID)
```

Comment:
did we still need to remove ` altcurrency.BAT.ToProbi`?

Reply:
thanks for the catch! reverted: e63fef4